snapcraft/main.py

```diff
@@ -110,6 +113,8 @@ def main():
 
     log.configure(log_level=log_level)
 
+    common.set_enable_parallel_builds(not args['--no-parallel-build'])
+
     try:
         commands.load(cmd).main(argv=args['ARGS'])
```

Comment:
so `<command>.main(project)` would work in most cases.

I correct myself; the `yaml` module already handle the plugins as it is part of the whole tree anyways.

This would potentially allow for parallelizing unit test run.